Create SUPPORT.md (#3340)
microsoft/winget-cli: SUPPORT.md

# Support

## How to file issues and get help  

This project uses [GitHub issues][gh-issue] to [track bugs][gh-bug] and [feature requests][gh-feature]. Please search the existing issues before filing new issues to avoid duplicates. For new topics, file your bug or feature request as a new issue.

For help and questions about using this project, please look at the [docs site for Windows Package Manager][docs] and our [Contributor's Guide][contributor] if you want to work on WinGet.

## Microsoft Support Policy  

> Support for Dev Home is limited to the resources listed above.

Reviewer: ```suggestion
Support for Windows Package Manager is limited to the resources listed above.
```

[gh-issue]: https://github.com/microsoft/winget-cli/issues/new/choose
[gh-bug]: https://github.com/microsoft/winget-cli/issues/new?assignees=&labels=Issue-Bug&projects=&template=Bug_Report.yml

> [gh-feature]: https://github.com/microsoft/winget-cli/issues/new?assignees=&labels=Issue-Bug&projects=&template=Feature_Request.yml

Reviewer 2: ```suggestion
[gh-feature]: https://github.com/microsoft/winget-cli/issues/new?assignees=&labels=Issue-Feature&projects=&template=Feature_Request.yml
```

[docs]: https://docs.microsoft.com/windows/package-manager

> [contributor]: https://github.com/microsoft/winget-cli/blob/main/CONTRIBUTING.md

Reviewer: ```suggestion
[contributor]: https://github.com/microsoft/winget-cli/blob/master/CONTRIBUTING.md
```
